src/TopbarSelect/TopbarSelect.tsx

```diff
@@ -38,24 +41,23 @@ const TopbarSelectValue = styled("div")`
   font-size: ${props => props.theme.font.size.fineprint}px;
   display: flex;
   align-items: center;
-  color: ${props => props.theme.color.text.dark};
-  & > :first-child {
-    margin-right: ${props => props.theme.space.element}px;
-  }
 `
 
-const TopbarSelectValueSpan = styled("span")<{ active?: boolean }>`
-  color: ${props => (props.active ? props.theme.color.text.dark : props.theme.color.text.lighter)};
+const TopbarSelectValueSpan = styled("span")`
+  margin-right: ${props => props.theme.space.element}px;
+`
+
+const TopbarSelectIcon = styled("div")`
+  paddingleft: ${props => props.theme.space.base}px;
```

Comment:
This is invalid CSS. I think you meant

```suggestion
  padding-left: ${props => props.theme.space.base}px;
```